tests/UserTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace App\Tests;
+
+use App\Entity\User;
+use PHPUnit\Framework\TestCase;
+
+class UserTest extends TestCase
+{
+    private User $user;
+
+    protected function setUp(): void
+    {
+        $this->user = new User();
+    }
+
+    public function testEmail(): void
+    {
+        $email = 'testemail@gmail.com';
+        $this->user->setEmail($email);
+        $this->assertEquals($email, $this->user->getEmail());
+        $this->assertEquals($email, $this->user->getUserIdentifier());
+    }
```

Comment:
This (and the others) are the one I think could be further improved upon : I initially wanted to implement "negative tests" (which would return an exeption, like if the user already exist, or if the email is nor valid), but as I use Symfony's `Validator/Constraint` to ensure this in my User entit, I'm really not sure how to implement it here